espnet2/asr/frontend/adapters_utils/example.py

```diff
@@ -0,0 +1,13 @@
+import s3prl.hub as hub
```

Comment:
Is this example helpful to understand the adapter usage? Maybe it is not necessary.

Reply:
I agree, `example.py` seems redundant after the revised README, I removed it.